### PR TITLE
add glidertest recipe

### PR DESCRIPTION
**Pull request by callumrollo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by callumrollo** [comment]
Hi @conda-forge/help-python looks like this is having the same CI issue as recently merged #28227

File: recipes/glidertest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% set name = "glidertest" %}
+{% set version = "0.0.1a6" %}
```
Comment (ocefpaf) [inline recipes/glidertest/meta.yaml:2]: Are you planning on a stable release anytime soon? If so, we can hold this off and publish that. If not you will have to follow cfep-05: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Comment (callumrollo) [reply recipes/glidertest/meta.yaml]: Yes! I did that in the wrong order. Stable v0.0.1 now on PyPI and used in this recipe

File: recipes/glidertest/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {% set name = "glidertest" %}
-{% set version = "0.0.1a6" %}
+{% set version = "0.0.1" %}
```
Comment (ocefpaf) [inline recipes/glidertest/meta.yaml:2]: :shipit:

**Comment by callumrollo** [comment]
The feedstock for this is still an empty repo :/ did something go wrong on the backend? https://github.com/conda-forge/glidertest-feedstock

**Comment by ocefpaf** [comment]
Should work as soon as https://github.com/conda-forge/staged-recipes/pull/28285 is on main.

**Pull request merged**